client/src/redux/actions/credentialsActions.js

```diff
@@ -24,11 +24,11 @@ const getCredentials = () => {
       .getCredentials()
       .then(success => {
         dispatch(getCredentialsSuccess(success));
+        dispatch(getCredentialsLoading(false));
       })
       .catch(error => {
         dispatch(getCredentialsError(true, error.message));
```

Comment:
Should `dispatch(getCredentialsLoading(false));` here as well.

Reply:
Indeed +1